utils/prometheus.go

```diff
@@ -0,0 +1,29 @@
+package utils
+
+import (
+	"github.com/FachschaftMathPhysInfo/altklausur-ausleihe/server/graph/model"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"gorm.io/gorm"
+)
+
+var (
+	// ExamsMarkedMetric is a prometheus metric for the amount of requested exams for watermarking
+	ExamsMarkedMetric = promauto.NewCounter(prometheus.CounterOpts{
```

Comment:
Why is this a gauge and not a counter?

Reply:
So you expect it to go down?

Reply:
No I dont, but it could, so counter is not a fitting metric type here.